lib/rubocop/cop/correctors/space_corrector.rb

```diff
@@ -12,12 +12,11 @@ class << self
         def empty_corrections(processed_source, corrector, empty_config,
                               left_token, right_token)
           @processed_source = processed_source
+          range = range_between(left_token.end_pos, right_token.begin_pos)
```

Comment:
@buehmann Thanks for the suggestion!
https://github.com/rubocop-hq/rubocop/compare/master...buehmann:empty-array-literal/7509